packages/oidc-client/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@aries-framework/oidc-client",
+  "main": "build/index",
+  "types": "build/index",
+  "version": "0.3.2",
+  "files": [
+    "build"
+  ],
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage": "https://github.com/hyperledger/aries-framework-javascript/tree/main/packages/oidc-client",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hyperledger/aries-framework-javascript",
+    "directory": "packages/oidc-client"
+  },
+  "scripts": {
+    "build": "yarn run clean && yarn run compile",
+    "clean": "rimraf -rf ./build",
+    "compile": "tsc -p tsconfig.build.json",
+    "prepublishOnly": "yarn run build",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@sphereon/openid4vci-client": "^0.3.4",
+    "class-transformer": "0.5.1",
+    "class-validator": "0.13.1",
+    "rxjs": "^7.2.0"
+  },
+  "peerDependencies": {
+    "@aries-framework/core": "0.2.5"
```

Comment:
I think this needs to be `0.3.2`?

Reply:
Good catch!

Reply:
I've moved them to `dependencies` in all packages as `peerDependencies` don't get updated. As we're alsways using the exact same versions there's not much benefit to using peer deps